cli_tools/gce_vm_image_import/importer/processor.go

```diff
@@ -51,10 +51,8 @@ func (d defaultProcessorProvider) provide(pd persistentDisk) (processors []proce
 		return
 	}
 
-	if d.ImportArguments.Inspect {
-		processors = append(processors, newDiskInspectionProcessor(d.diskInspector, d.ImportArguments))
-		processors = append(processors, newUefiProcessor(d.computeClient, d.ImportArguments))
-	}
+	processors = append(processors, newDiskInspectionProcessor(d.diskInspector, d.ImportArguments))
```

Comment:
Also update this so that inspection failure doesn't terminate the import:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/897cdfe3a16467bcb2842d127d137476851025e5/cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go#L41

Reply:
done